change/@internal-react-composites-93f823de-fb23-4096-aa20-ffb9254e29e3.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Update Feature Toggle API",
```

Comment:
Would be good to note here the breaking change that developers need to flip from hiddenElements to Options